angular/src/app/keyword-mapping/keyword-mapping.component.html

```diff
@@ -0,0 +1,28 @@
+<h3> Detect Bias </h3>
+<br>
+<div> 
+	<input type="text"
+	name="keywords"
+	[(ngModel)]= "userInput.keywordString">	
+	<input type="text"
+	name="left-axis"
+	[(ngModel)]= "userInput.leftString">
+	<input type="text"
+	name="right"
+	[(ngModel)]= "userInput.rightString">
+</div>
+<button
+	class="btn btn-primary"
```

Comment:
you're using indentation for two different things here. It is convention to use it to show that there's a new element, not a new property.

```
<button property1="1", property2="property>
  buttonlabel
</button>
```

Reply:
thanks